bin/olly.ml

```diff
@@ -35,6 +36,7 @@ let print_percentiles json output hist =
     |]
   in
   let oc = match output with Some s -> open_out s | None -> stderr in
+  let total_time = Unix.gettimeofday () -. start_time in
```

Comment:
Total time should be computed from the trace? The start of `olly` and the start of the program may not align. There is an explicit event at the beginning of the trace IINM @sadiqj?

Reply:
https://github.com/ocaml/ocaml/blob/b2f93feec731f40ee0e8ea2d4da048ebb6d31ca3/otherlibs/runtime_events/runtime_events.ml#L77-L78

Reply:
Good point, updated.